src/runtime_src/tools/xclbinutil/SectionDebugIPLayout.cxx

```diff
@@ -109,6 +111,9 @@ SectionDebugIPLayout::getDebugIPType(std::string& _sDebugIPType) const {
   if ( _sDebugIPType == "AXI_STREAM_PROTOCOL_CHECKER" ) 
     return AXI_STREAM_PROTOCOL_CHECKER;
 
+  if ( _sDebugIPType == "ACCEL_DEADLOCK_DETECTOR" ) 
+    return AXI_STREAM_PROTOCOL_CHECKER;
```

Comment:
this looks incorrect

Reply:
Thanks Paul.
It looks like my merge missed this pull request.  I will merge it shortly.